### PR TITLE
feat: added MiniCalendar.jsx to src/components

### LINKED ISSUE #136: UI: Create MiniCalendar Component
## Feature Description

- Create mini-calendar component, using [Mantine](https://mantine.dev) as a starting point.

## Definition of Done

- _No need to style the component or place it on the page, simply create the component._ Styling will be handled by #138. 

- [x] Calendar component created in /src/components/MiniCalendar.jsx
- [x] Component must be functional and ready to be dropped into the project.

## Prototype
image

## Comments/Questions

This is a screenshot of the Mantine module you'll be using, from the Mantine documentation. 

[Mantine Calendar Docs](https://mantine.dev/dates/calendar/)

image

### PR DESCRIPTION
**Pull request by Doug-Colin**
## Pull Request type

- [x] Feature

## What is the current behavior?

Clicking calendar displays "One day, I will be a calendar."

Issue Number: Closes #136

## What is the new behavior?

- No new behavior; component is not yet active. 

## Does this introduce a breaking change?

- [ ] No

### CODE REVIEW

File: src/components/MiniCalendar.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { useState } from 'react'
+import { Calendar } from '@mantine/dates'
+
+function MiniCalendar() {
```
Comment (JoeKarow) [inline src/components/MiniCalendar.jsx:4]: this needs to be exported:

either use 
```js
export function MiniCalendar() {
```
or
```js
export const MiniCalendar = () => {
``` 
if you want to do it all arrow function like.

**Review by JoeKarow — COMMENTED** [review]
If you want to preview your component, you can create a temporary file in `src/pages` called something like `test.jsx`, import and display your component on it and then you can view it at `http://localhost:3000/test`

File: src/components/MiniCalendar.jsx (hunk 1 above)
Comment (Doug-Colin) [reply src/components/MiniCalendar.jsx]: Thanks! Fixed it and ensured functionality.

**Review by JoeKarow — APPROVED** [review]

**Pull request merged**